.github/workflows/ci.yml

```diff
@@ -2,17 +2,19 @@ name: CI
 on:
   pull_request:
   push:
+    branches:
+      - master
```

Comment:
- https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpull_requestpull_request_targetbranchesbranches-ignore